glui_clipboard.cpp

```diff
@@ -0,0 +1,303 @@
+/****************************************************************************
+
+  GLUI User Interface Toolkit
+  ---------------------------
+
+     glui_clipboard_x11.cpp - GLUI_Clipboard_Object class
+
+	 This file -is- in the public domain. Circa summer 2017.
```

Comment:
The program is copyrighted. I just don't want to mark up my contributions. These boiler plates are really just cataloging the history of the software. Public domain is a superset of the wider license, so  I do not think it's anything to worry about. What I want to communicate is feel free to use this code to implement copy-and-paste (which of course can be done regardless but I want to encourage it freely without any possessive overtones.) 

You can change anything when committing these changes. I generally won't jump through hoops myself because I have a lot of work to do always and I feel like contributions should be received in appreciation and carefully integrated and that contributors should be thanked and not browbeaten with additional workloads :)